lisa/wa.py

```diff
@@ -250,11 +252,18 @@ def _needs_params(col):
         """
         parameters = list(inspect.signature(col).parameters.items())
         return any(
-            param.default == param.empty
+            param.default is param.empty and param.kind not in (param.VAR_POSITIONAL, param.VAR_KEYWORD)
             # The first parameter is provided so we can skip it
             for name, param in parameters[1:]
         )
 
+    @property
```

Comment:
This job discovery is now duplicated with `_get_df`, they need to be unified. It's probably possible to rewrite `_get_df` in terms of that `jobs` property so we don't accidentally start having diverging definition of the set of valid jobs for both uses.